gdunit4_testadapter/TestAdapterRunner.cs

```diff
@@ -0,0 +1,7 @@
+namespace GdUnit4.TestAdapter;
+
+public partial class TestAdapterRunner : Api.TestRunner
```

Comment:
Do not add the `TestAdapterRunner` files to your repository!
These files are generated and the folder `gdunit_testadapter` should be added to the .gitignore file

Reply:
> the folder gdunit_testadapter should be added to the .gitignore file

@MikeSchulze  Can this be automated?

Reply:
This has been resolved in https://github.com/Pieeer1/GodotVoipNet/pull/13